pkg/memtier/prompt.go

```diff
@@ -46,7 +46,7 @@ type Prompt struct {
 	pidwatcher   PidWatcher
 	mover        *Mover
 	pages        *Pages
-	aranges      *AddrRanges
+	adranges     *AddrRanges
```

Comment:
There is a command `arange` in the prompt that is connected to contents of the `aranges` variable. I don't think we should change the name of the variable and break this connection.